contrib/grinder/scripts/annotationsingest.py

```diff
@@ -0,0 +1,86 @@
+import random
+try: 
+  from com.xhaus.jyson import JysonCodec as json
+except ImportError:
+  import json
+from utils import *
+from net.grinder.script import Test
+from net.grinder.plugin.http import HTTPRequest
+
+class AnnotationsIngestThread(AbstractThread):
+  # The list of metric numbers for all threads in this worker
+  annotations = []
+  
+  # Grinder test reporting infrastructure
+  test1 = Test(2, "Annotations Ingest test")
+  request = HTTPRequest()
+  test1.record(request)
+
+
+  @classmethod
+  def create_metrics(cls, agent_number):
+    """ Generate all the metrics for this worker
+
+    The metrics are a list of batches.  Each batch is a list of metrics processed by
+    a single metrics ingest request.
+    """
+    annotations =  generate_metrics_tenants(default_config['num_tenants'],
+                                            default_config['annotations_per_tenant'], agent_number,
+                                            default_config['num_nodes'], 
+                                            cls.generate_annotations_for_tenant)
+
+    cls.annotations = cls.divide_annotations_into_batches(annotations, 1)
```

Comment:
This is a bit wierd, creating a batch size of 1.  The batches serve no purpose in this module and should be eliminated.